### PR TITLE
Improved the SearchableDropdown component

### PR DESCRIPTION
**Pull request by ivorpad**

### CODE REVIEW

File: src/styles/dotcms-theme/components/_calendar.scss [hunk 1]
```diff
@@ -75,8 +75,8 @@
     font-weight: 500;
```
Comment (alfredo-dotcms) [inline src/styles/dotcms-theme/components/_calendar.scss:75]: `$font-weight-semi-bold`

File: src/styles/dotcms-theme/components/_dataview.scss [hunk 2]
```diff
@@ -14,29 +18,34 @@
...
     border-width: 0 0 1px 0;
 }
+
 .p-dataview .p-dataview-footer {
     background: #ffffff;
     color: #000001;
     border: 1px solid #e4e4e4;
     border-width: 0 0 1px 0;
     padding: 0.75rem 0.75rem;
     font-weight: 500;
```
Comment (alfredo-dotcms) [inline src/styles/dotcms-theme/components/_dataview.scss:40]: $font-weight-semi-bold

File: src/styles/dotcms-theme/components/_dataview.scss [hunk 3]
```diff
@@ -14,29 +18,34 @@
...
```
Comment (alfredo-dotcms) [inline src/styles/dotcms-theme/components/_dataview.scss:19]: $font-weight-semi-bold

File: src/styles/dotcms-theme/components/_dataview.scss [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
     border-radius: 0;
 }
+
 .p-dataview .p-paginator-bottom {
     border-width: 0 0 1px 0;
     border-radius: 0;
 }
+
 .p-dataview .p-dataview-header {
     background: #ffffff;
```
Comment (alfredo-dotcms) [inline src/styles/dotcms-theme/components/_dataview.scss:14]: `$white`

File: src/styles/dotcms-theme/components/_dataview.scss [hunk 5]
```diff
@@ -14,29 +18,34 @@
     padding: 0.75rem 0.75rem;
     font-weight: 500;
 }
+
 .p-dataview .p-dataview-content {
     background: #ffffff;
```
Comment (alfredo-dotcms) [inline src/styles/dotcms-theme/components/_dataview.scss:23]: `$white`

File: src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.html [hunk 6]
```diff
@@ -14,9 +14,8 @@
         *ngIf="label"
         (click)="searchPanel.toggle($event)"
         [label]="label"
-        icon="fa fa-caret-down"
+        icon="pi pi-chevron-down"
```
Comment (alfredo-dotcms) [inline src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.html:17]: test for then new icon an update removing the css class

Comment (ivorpad) [reply src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.html:17]: What do you mean @alfredo-dotcms ?

**Review by fmontes — APPROVED** [review]

**Review by fmontes — CHANGES_REQUESTED** [review]

**Pull request merged**